### ISSUE TITLE
[Mocha] Crash on testFinish with errors

### ISSUE THREAD

**Issue by itsJess1ca** [buildkite/test-collector-javascript#57]
Hello,
I'm in the process of implementing BuildKite as a trial for my team, however ran into the following issue when running our Mocha suite in a Github Action. 

```
Unhandled asynchronous exception: TypeError: Cannot read properties of undefined (reading 'map')
    at failureExpanded (/home/runner/work/my-repo/my-repo/node_modules/buildkite-test-collector/util/failureExpanded.js:2:17)
    at MochaBuildkiteAnalyticsReporter.testFinished (/home/runner/work/my-repo/my-repo/node_modules/buildkite-test-collector/mocha/reporter.js:59:27)
    at RunnerMain.<anonymous> (/home/runner/work/my-repo/my-repo/node_modules/buildkite-test-collector/mocha/reporter.js:34:14)
    at RunnerMain.emit (node:events:525:35)
    at RunnerMain.emitSubprocessEvents (/home/runner/work/my-repo/my-repo/node_modules/mocha-parallel-tests/dist/main/runner.js:158:30)
    at RunnerMain.reEmitSubprocessEvents (/home/runner/work/my-repo/my-repo/node_modules/mocha-parallel-tests/dist/main/runner.js:54:14)
    at TaskManager.<anonymous> (/home/runner/work/my-repo/my-repo/node_modules/mocha-parallel-tests/dist/main/mocha.js:77:20)
    at TaskManager.emit (node:events:513:28)
    at TaskManager.onTaskProcessingFinished (/home/runner/work/my-repo/my-repo/node_modules/mocha-parallel-tests/dist/main/task-manager.js:22:18)
    at TaskManager.emit (node:events:513:28)
```

Note: this is not blocking me as I've switched to JUnit upload, however figured I'd raise the issue for investigation 👍

**Comment by amybiyuliu** [comment]
Thanks @itsJess1ca for raising this issue! I've created an internal ticket and we'll look into it.

**Comment by KaoruDev** [comment]
Hihi, I've encountered the same issue. We're using mocha `8.1.3` with Cypress version `12.5.1`.

**Comment by nprizal** [comment]
This issue has been resolved in [v1.5.0](https://github.com/buildkite/test-collector-javascript/releases/tag/v1.5.0)
